### ISSUE TITLE
Inbox says "1 Unread Message", clicking it shows no message

### ISSUE THREAD

**Issue by dginovker** [LemmyNet/lemmy-ui#810]
### Issue Summary

Inbox says "1 Unread Message", clicking it shows no message

### Steps to Reproduce

I do not have a repro at this time

### Technical details

On the current live version of Lemmy.ml (BE: 0.16.7)

Gif of the issue:

![lemmy in a nutshell](https://user-images.githubusercontent.com/32943174/193036353-78a69bac-162e-4f81-8bc5-e274c3b92f33.gif)

Potentially useful console screenshots:

![image](https://user-images.githubusercontent.com/32943174/193037103-4d44a472-4645-49f4-9338-ebe9f8e5bad4.png)
![image](https://user-images.githubusercontent.com/32943174/193037214-74d5f8d5-09c7-4fae-96a7-ae7e34f8d5f8.png)

**Comment by Nutomic** [comment]
Im also experiencing wrong number of notifications shown on lemmy.ml since some days. It always seems to be one more than the actual number.

**Comment by dessalines** [comment]
One issue, is that when you reply to a comment, the back-end automatically marks it as read, but I need to make sure this is also done in the UI.

There might be a few other cases like that too, such as clicking on a comment toast or link. 

If these cases get really complex, I might just do the simple but wasteful thing of periodically fetching from the `GetUnreadCount` endpoint, which currently is only run on initial page load.

**Comment by Nutomic** [comment]
No, the number is still showing wrong after page reload, or after new login. So it seems to be calculated/stored wrong in the database.

**Comment by dessalines** [comment]
I *think* I might know what the problem is, the back end isn't checking for deleted comments. 

I need to see the output of the `GetUnreadCount` line, to verify that this is a backend issue.
